service-runner/src/main/resources/version-info.properties

```diff
@@ -1 +1 @@
-version=2.10.2
\ No newline at end of file
+version=2.10.2R
```

Comment:
What is the `R`?

Reply:
Thanks for catching the accidental key stroke.